samples/event-sourcing-esdb-simple/src/test/java/io/eventdriven/ecommerce/api/ECommerceApplicationTests.java

```diff
@@ -0,0 +1,13 @@
+package io.eventdriven.ecommerce.api;
+
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.context.SpringBootTest;
+
+@SpringBootTest
+class ECommerceApplicationTests {
+
+    @Test
+    void contextLoads() {
```

Comment:
#minor
No tests,  so hard to say what is going here. Good tests are also good documentation :)

Reply:
I added a whole test suite for API tests. See more in `ApiSpecification` and related API tests. I also added GH action to run it. 🙂